cibyl/models/ci/build.py

```diff
@@ -31,16 +31,27 @@ class Build(Model):
             'arguments': [Argument(name='--build-status', arg_type=str,
                                    description="Build status")]
         },
+        'duration': {
+            'attr_type': int,
+            'arguments': [],
+        },
     }
 
-    def __init__(self, build_id: str, status: str = None):
-        super().__init__({'build_id': build_id, 'status': status})
+    def __init__(self, build_id: str, status: str = None,
+                 duration: int = None):
+        if duration and duration > 10000:
```

Comment:
I think this can be confusing, since we only perform a change of units in some cases, but we do not keep track of which unit of time is used

Reply:
Yeah, you're right. Any suggestions on how to track it?

Reply:
I would store it in a specific unit (seconds seems like a reasonable choice) and then we can choose to simply print it in seconds or use a more complex representation when printing it, using seconds or minutes depending on the magnitude

Reply:
k, sounds reasonable. removed it from here